testmod/src/main/java/us/potatoboy/ledger/testmod/mixin/ClientPlayerInteractionManagerMixin.java

```diff
@@ -0,0 +1,33 @@
+package us.potatoboy.ledger.testmod.mixin;
```

Comment:
Should be in src/testmod/java/us/potatoboy/ledger/testmod/mixin

Reply:
Huh?
Do you mean I should put it in the `testmod` source set of the `testmod` subproject?